code/modules/library/lib_items.dm

```diff
@@ -316,3 +316,49 @@
 	else
 		user << "<font color=red>No associated computer found. Only local scans will function properly.</font>"
 	user << "\n"
+
+/obj/item/weapon/chisel
+	name = "chisel"
+	desc = "Leave informative messages for the crew, including the crew of future shifts!"
+	icon = 'icons/obj/items.dmi'
+	icon_state ="chisel"
+	throw_speed = 3
+	throw_range = 5
+	w_class = WEIGHT_CLASS_TINY
+
+/obj/item/weapon/chisel/afterattack(atom/target, mob/user, proximity)
+	if(!proximity)
+		return
+	if(istype(target, /turf/open/floor) || istype(target, /turf/closed/wall))

```

Comment:
also you'll need something to make sure chisel messages don't spawn on turfs that are.... not floors or walls